canonical/charmhub.io
# fix: disable search on home page while charms are in loading state

## Done
- Disables search bar and button on home page while charms / bundles are loading - search was locking up if you entered something before all charms were loaded
- Adds test for new functionality
- Note: this should be a temporary fix whilst we investigate performance and try to improve the load time of `/store.json`

## How to QA
- Go to https://charmhub-io-2032.demos.haus/ and while charms are loading on initial load, make sure search bar is disabled
- Search bar should also be disabled when filters are selected and charms are loading again

## Testing

## Issue / Card
Fixes [WD-13296](https://warthogs.atlassian.net/browse/WD-13296) and [1880](https://github.com/canonical/charmhub.io/issues/1880)

[WD-13296]: https://warthogs.atlassian.net/browse/WD-13296?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
